CodePush.m

```diff
@@ -216,38 +203,35 @@ - (void)saveFailedUpdate:(NSString *)packageHash
 }
 
 /*
+ * This method is called in notifyApplicationReady to register the fact that
+ * the pending update succeeded and therefore can be removed.
```

Comment:
I don't think you should include caller information in comments.  Just state what this method does.